examples/usecases/transformers-next-item-prediction-with-pretrained-embeddings.ipynb

```diff
@@ -0,0 +1,5658 @@
+{
```

Comment:
<p>I do not think we should use workflow outputs from NVTabular. This would not scale to a large dataset. I think we should store them to disk via to_parquet.</p><p> </p><p>Users will adopt our examples to their dataset and it is likely they would run out of memory?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>change made</p>